cognite/client/data_classes/capabilities.py

```diff
@@ -1128,8 +1140,40 @@ class Scope:
         All = AllScope
 
 
+@dataclass
+class LegacyModelHostingAcl(LegacyCapability):
+    _capability_name = "modelHostingAcl"
+    actions: Sequence[Action]
+    scope: AllScope = field(default_factory=AllScope)
+
+    class Action(Capability.Action):
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+
+
+@dataclass
+class LegacyGenericsAcl(LegacyCapability):
+    _capability_name = "genericsAcl"
+    actions: Sequence[Action]
+    scope: AllScope
+
+    class Action(Capability.Action):
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+
+
 _CAPABILITY_CLASS_BY_NAME: MappingProxyType[str, type[Capability]] = MappingProxyType(
-    {c._capability_name: c for c in Capability.__subclasses__() if c is not UnknownAcl}
+    {
+        c._capability_name: c
+        for c in itertools.chain(Capability.__subclasses__(), LegacyCapability.__subclasses__())
+        if c is not UnknownAcl and c is not LegacyCapability
```

Comment:
This should be an `or`?
```suggestion
        if c not in (UnknownAcl, LegacyCapability)
```